gz_ros2_control/gz_hardware_plugins.xml

```diff
@@ -0,0 +1,10 @@
+<library path="gz_hardware_plugins">
+  <class
```

Comment:
Would it be possible to use the old plugin name as well? Otherwise, people will 
have to be forced to update the URDF/xacro files.

Reply:
Adding this should help:
```xml
 <class
    name="ign_ros2_control/IgnitionSystem"
    type="gz_ros2_control::GazeboSimSystem"
    base_class_type="gz_ros2_control::GazeboSimSystemInterface">
    <description>
      GazeboPositionJoint
    </description>
  </class>
```